tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -334,7 +334,7 @@ public function contextSelectionProvider(): array
             ],
         ];
         $form = new Form();
-        $custom = $this->getMockBuilder('Cake\View\Form\ContextInterface')->getMock();
+        //$custom = $this->getMockBuilder('Cake\View\Form\ContextInterface')->getMock();
```

Comment:
How do we address this? Providers being static means can't get the mock builder now.